tests/200-py-unittest.sh

```diff
@@ -39,9 +39,12 @@ fi
 
 TOXENVS="py27"
 if command -v python3.5 &>/dev/null; then
-	TOXENVS="${TOXENVS},py35"
 	if command -v pyenv &>/dev/null; then
```

Comment:
As someone who does not understand the python environment well, I would say it would be nice to document what is being checked here.

Reply:
I agree that an explaining comment would be nice. But would not even block on this.

Reply:
I'd like to do that in a followup pr tho.